File: samples/node/get-module-answers.js
const axios = require('axios');

axios({
  method: 'get',
  baseURL: 'http://api.beepquest.com/',
  url: '/v1/question-module-answers',
  headers: {
    'BQAPPTOK': '{APP_TOKEN}',
    'BQMODTOK': '{MODULE_TOKEN}',
  },
  params: {
    'initialDate': '2018-01-01',
    'limit': '10',
    'users': '.com,,,,ivan@beepquest.com',
  }
}).then((response) => {
  console.log(response.data);
})

File: samples/python3/get_module_answers.py
import requests

headers = {
    'BQAPPTOK': '{BQAPPTOK}',
    'BQMODTOK': '{BQMODTOK}',
}
payload = {
    'initialDate': '2018-01-01',
    'limit': '10',
    'users': 'ivan@beepquest.com,fernando@hellodave.mx',
}
r = requests.get('{url_api}/v1/question-module-answers'.format(
    url_api='http://api.beepquest.com'
), headers=headers, params=payload)

print(r.json())
